data/test/scenarios/wml_tests/FilterWML/special_id_active_lua_function.cfg

```diff
@@ -0,0 +1,115 @@
+#textdomain wesnoth-test
+
+#####
+# This unit test reproduces setup of weapon special such as https://github.com/inferno8/wesnoth-Era_of_Magic/blob/adf227cba2e66602c4bb49dc203fa751865aa124/utils/beam.cfg, which is not supposed to work in rpg.
+##
+# API(s) being tested: [event][filter_attack]special_id_active, weapon special [filter_self]lua_function
```

Comment:
To fit with the other tests, this line should be, without a space after the comma:
`[event][filter_attack]special_id_active=,[damage][filter_self]lua_function=`

With @Pentarctagon's test reorg, his aim was to reduce the number of APIs listed in that line. However, I can see the logic of saying that this is specifically testing that the event filtering code has a working lua filter, whether or not that filtering code is shared with other weapon filters.

Reply:
More generally, the goal is for unit tests to be primarily testing only one thing (or as few things as possible) even if they depend on other things also working.